### PR TITLE
S1t5 styling

### PR DESCRIPTION
**Pull request by emferrari31**
Here I've styled my site and made is responsive

### CODE REVIEW

File: collectionApp.css
Comment (ramsey-merdassi) [inline collectionApp.css:1]: Rename html file to index.php and consider separating 'connect to database' code in separate db file using require_once.
Consider adding your images as a field in SQL using urls.
Better to use margin around images and separate html text tags than breaks.

Comment (ramsey-merdassi) [reply collectionApp.css:1]: Other than that all good

File: collectionApp.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    .imageAndTextContainer {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+
+    .imageContainer {
+        flex-direction: row;
+    }
```
Comment (matt-ioa) [inline collectionApp.css:49]: Since you haven't used `display: flex` anywhere on this class, this won't have an effect

File: collectionApp.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    .textContainer {
+        flex-container: row;
```
Comment (matt-ioa) [inline collectionApp.css:52]: There is no `flex-container` property - did you mean `flex-direction`?

File: index.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    global $db;
+    $query = $db->prepare("SELECT `brandName`, `model`, `type`, `megapixels`, `weight(g)`, `images` FROM `cameras`");
+    $query->execute();
+    $results = $query->fetchAll();
+    
+    foreach ($results as $result) {
+        echo '<div class="imageAndTextContainer">';
+        echo '<div class = "imageContainer">';
+        echo '<img class="imagesClass" src="' . $result['images'] . '" alt="camera">';
+        echo '<br><br>';
```
Comment (matt-ioa) [inline index.php:25]: Rather than using `<br>` tags, it's usually better to add padding/margin underneath the image tag

File: index.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        echo '</div>';
+
+        echo '<div class = "textContainer">';
+        echo $result['brandName'] . " <br>" . $result['model'] . " <br>" . $result['type'] . " <br>" .
+            $result['megapixels'] . " megapixels<br>" . $result['weight(g)'] . "g<br><br>";
```
Comment (matt-ioa) [inline index.php:30]: Again, putting each bit of information in a `div` or `p` tag will give you more control over padding and margin than using `<br>`, but in this case it's up to you whether or not to leave it.

**Pull request merged**